assets/properties-panel.css

```diff
@@ -10,6 +10,8 @@
   --color-red-360-100-97: #fff0f0;
```

Comment:
The indentation line (left) for the nested entries is not long enough. It should be fit to the last entry's input.

![image](https://user-images.githubusercontent.com/9433996/132176561-00e7f54f-1b9a-44d1-8f66-694644b2c46e.png)

Reply:
![image](https://user-images.githubusercontent.com/28307541/132218443-46c982cf-c721-4149-b4e4-71cf3dffd109.png)

Implemented via https://github.com/bpmn-io/properties-panel/pull/93/commits/eca57f97052a401de8a702856d36d0d1409da1b8